src/test/java/uk/gov/pay/ledger/transaction/dao/TransactionDaoSearchIT.java

```diff
@@ -137,6 +137,7 @@ public void shouldReturn1Record_whenSearchingByReference() {
 
         TransactionSearchParams searchParams = new TransactionSearchParams();
         searchParams.setAccountId(gatewayAccountId);
+        searchParams.setExactReferenceMatch(true);
```

Comment:
May be rename method name to `shouldReturn1Record_whenSearchingByExactReference`